website/docs/r/api_token.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_api_token"
+sidebar_current: "docs-vcfa-resource-api-token"
+description: |-
+  Provides a resource to manage API Tokens. API Tokens are an easy way to authenticate to VCFA. 
+  They are user-based and have the same role as the user.
+---
+
+# vcfa\_api\_token 
+
+Provides a resource to manage API Tokens. API Tokens are an easy way to authenticate to VCFA. 
+They are user-based and have the same role as the user.
+
+## Example usage
+
+```hcl
+resource "vcfa_api_token" "example_token" {
+  name             = "example_token"
+  file_name        = "example_token.json"
+  allow_token_file = true
+}
+```
+
+## Argument reference
+
+The following arguments are supported:
+
+* `name` - (Required) The unique name of the API Token for a specific user.
+* `file_name` - (Required) The name of the file which will be created containing the API Token
```

Comment:
Maybe we could provide small hint where to get this token from?

Reply:
Like a link to here:
https://github.com/vmware/terraform-provider-vcfa/pull/22/files#diff-4839085b71de83d7294472f5f8fa6652faf4b0b24df211c8eaf945ac96933aa5R96-R97

Reply:
Added an explanation of JSON file contents